uldk_gugik.py

```diff
@@ -839,34 +821,25 @@ def performRequestParcel(self, region, parcel, teryt=None, zoomToFeature=True):
                 layer.updateFields()
 
                 layer.updateFields()
-                counter = 1
-
-            idx = layer.fields().indexFromName('identyfikator')
-            attrMap = {counter: {idx: teryt}}
-            provider.changeAttributeValues(attrMap)
-
-            voiv = layer.fields().indexFromName('województwo')
-            attrMap = {counter: {voiv: voivodeship.rstrip()}}
-            provider.changeAttributeValues(attrMap)
-
-            if parcel is not None:
-                par = layer.fields().indexFromName('numer')
-                attrMap = {counter: {par: parcel}}
-                provider.changeAttributeValues(attrMap)
-
-            if region is not None:
-                reg = layer.fields().indexFromName('obręb')
-                attrMap = {counter: {reg: region}}
-                provider.changeAttributeValues(attrMap)
-            if commune is not None:
-                com = layer.fields().indexFromName('gmina')
-                attrMap = {counter: {com: commune}}
-                provider.changeAttributeValues(attrMap)
-
-            if county is not None:
-                con = layer.fields().indexFromName('powiat')
-                attrMap = {counter: {con: county}}
-                provider.changeAttributeValues(attrMap)
+
+            feat = QgsFeature(provider.fields())
+            geom = QgsGeometry().fromWkt(wkt)
+            projectCrs = str(2180)
+            if str(projectCrs) != '2180':
+                sourceCrs = QgsCoordinateReferenceSystem.fromEpsgId(int(projectCrs))
+                destCrs = QgsCoordinateReferenceSystem.fromEpsgId(2180)
```

Comment:
Jakby CRS był ściągany z projektu to ten projectCrs mógłby zostać, ale dest CRS bym też zostawił wtedy na project CRS bo jednak dobrze by było żeby geodeta wybrał sobie działkę z układu np. 2000 i innych a nie tylko PL-92